src/etc/githook/commit-msg

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# To apply spotless for every commit copy this file to folder .git/hooks/
+# or add below line to existing .git/hooks/commit-msg
```

Comment:
In my opinion `.git/hooks/commit-msg` is not a good place for this check. Lets look at samples:
```
cat .git/hooks/commit-msg.sample | head -5
#!/bin/sh
#
# An example hook script to check the commit log message.
# Called by "git commit" with one argument, the name of the file
# that has the commit message.  The hook should exit with non-zero
```

`pre-commit` seeds to be more suitable:
```
cat .git/hooks/pre-commit.sample | head -5
#!/bin/sh
#
# An example hook script to verify what is about to be committed.
# Called by "git commit" with no arguments.  The hook should
# exit with non-zero status after issuing an appropriate message if
```